ab_testing_tool/settings/test_settings.py

```diff
@@ -7,3 +7,12 @@
 if not ENV_SETTINGS.get('enable_test_logging'):
     for logger in LOGGING['loggers']:
         LOGGING['loggers'][logger]['handlers'] = ['null']
+
+# make tests faster
+DATABASES = {
```

Comment:
I feel like I got burned a little while ago for suggesting we use sqlite for local development until we start doing something supported by postgres... either way, I think you could accomplish the same effect with:
```
DATABASES['default']['ENGINE'] = 'django.db.backends.sqlite3'
```